### PR TITLE
intial integration redis-rs

### PR DESCRIPTION
**Pull request by commented-line**
Redis-rs has 5 million downloads 12k downloads per day. Redis-rs is the most popular redis library for rust. It is also used as dependecy for many rust libraries.

### CODE REVIEW

**Comment by commented-line** [comment]
@Navidem @jonathanmetzman need approval for build tests and merge

**Comment by Navidem** [comment]
Waiting for the panel to approve the merge.

File: projects/redis-rs/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+FROM gcr.io/oss-fuzz-base/base-builder-rust as builder
+
+## Install build dependencies.
+RUN apt-get update
```
Comment (jonathanmetzman) [inline projects/redis-rs/Dockerfile:18]: There's no point in this line.

Comment (code-terror) [reply projects/redis-rs/Dockerfile]: Removed the line..

**Review by Navidem — APPROVED** [review]
LGTM

**Pull request merged**